cnames_active.js

```diff
@@ -1045,6 +1045,7 @@ var cnames_active = {
   "endpoint-imposter": "lukaszmakuch.github.io/endpoint-imposter",
   "energy": "energychain.github.io/energy",
   "englishell": "tzador.github.io/englishell",
+  "enn": "https://loganpaxton.github.io/enn/",
```

Comment:
```suggestion
  "enn": "loganpaxton.github.io/enn",
```
As the CI error indicates, you do not need the `https://` prefix, nor the trailing slash